pyproject.toml

```diff
@@ -77,14 +71,15 @@ markers = [
 addopts = "--cov=udata_hydra/ --cov-report term-missing"
 
 [tool.ruff]
-lint = { extend-select = ["I"], ignore = ["F401"] } # ["I"] is to also sort imports with an isort rule # TODO: remove ignore F401 later
+lint = { extend-select = ["I"], ignore = ["F401"] }
+# ["I"] is to also sort imports with an isort rule
+# TODO: remove ignore F401 later
 line-length = 100
```

Comment:
FYI, on udata we have a `ruff.toml` file which does the job quite well imho:

```toml
line-length = 100

[lint]
extend-select = ["I"] # also sort imports with an isort rule
```

Reply:
I still have some issues with imports in `__init__.py` files which are flagged by Ruff if I don't ignore the F401 rule. If you have suggestions on how to solve that I'm all ears!

Reply:
The [ruff docs](https://docs.astral.sh/ruff/rules/unused-import/) have a few ideas to fix it. If you still feel uncomfortable about those fixes, I guess there are two other options left (if you want to get rid of the `ignore = ["F401"]` rule):
1. explicitly disable it with an inline comment `# noqa: F401`
2. explicitly disable it for the whole module `# ruff: noqa: F401`

I'm fine with keeping the rule ignore in the config though, so it's your call ;)

Reply:
Great, thank you for the info. I changed the behavoir. t's indeed better!